test/e2e/play_kube_test.go

```diff
@@ -2685,7 +2685,7 @@ spec:
 		err := generateKubeYaml("pod", pod, kubeYaml)
 		Expect(err).ToNot(HaveOccurred())
 
-		kube := podmanTest.Podman([]string{"play", "kube", kubeYaml})
+		kube := podmanTest.Podman([]string{"play", "kube", "--expose", kubeYaml})
```

Comment:
Could you add an inspect afterwards to make sure the ports are exposed?

Reply:
That is already part of the test few lines below.